.github/workflows/container-image.yml

```diff
@@ -147,83 +165,27 @@ jobs:
       - name: Test the server image
         run: LOCAL_TAG=samba-server:${{ env.IMG_TAG }} tests/test-samba-container.sh
 
-  # Reminder: the nightly-server images consume nightly samba rpm builds
-  # it is not *just* an image that gets built nightly
-  build-nightly-server:
-    strategy:
-      matrix:
-        os: [centos, fedora]
-        arch: [amd64]
-    runs-on: ubuntu-latest
-    env:
-      BUILDAH_FORMAT: oci
-      IMG_TAG: nightly-${{ matrix.os }}-${{ matrix.arch }}
-    steps:
-      - uses: actions/checkout@v3
-      - name: Build the nightly server image
-        run: make KIND=server PACKAGE_SOURCE=nightly OS_NAME=${{ matrix.os }} BUILD_ARCH=${{ matrix.arch }} build-image
-      - name: Upload nightly server image
-        uses: ishworkh/container-image-artifact-upload@v1.0.0
-        with:
-          image: "samba-server:${{ env.IMG_TAG }}"
-          container_engine: ${{ env.CONTAINER_CMD }}
-          retention_days: 1
-
-  build-nightly-ad-server:
```

Comment:
shouldn't this be removed due to the introduction of `build-ad-server(nightly, *, *)`

Reply:
withdrawinbg my change request as it referred to a non-issue